images/openshift-ci-azure/Dockerfile

```diff
@@ -0,0 +1,28 @@
+# This Dockerfile is used by openshift CI
+
+FROM registry.ci.openshift.org/openshift/release:golang-1.16 AS builder
+WORKDIR /go/src/github.com/code-ready/crc
+COPY . .
+RUN make release
+
+FROM quay.io/centos/centos:stream8
+COPY --from=builder /go/src/github.com/code-ready/crc /opt/crc
+COPY --from=builder /go/src/github.com/code-ready/crc/out/windows-amd64/crc.exe /bin/crc.exe
+COPY --from=builder /go/src/github.com/code-ready/crc/images/openshift-ci/mock-nss.sh /bin/mock-nss.sh
+COPY --from=builder /go/src/github.com/code-ready/crc/images/openshift-ci-azure/azure-cli.repo /etc/yum.repos.d/azure-cli.repo
+
+
+RUN yum update -y && \
+    yum install --setopt=tsflags=nodocs -y \
```

Comment:
Why the `tsflags=nodocs`?

Reply:
To make sure we don't install the doc for the packages we install to keep the image size minimal.

Reply:
But is the image size that constrained? This feels like premature optimization :)